Menu/Provider/GroupMenuProvider.php

```diff
@@ -147,9 +147,9 @@ public function get($name, array $options = array())
                 if (isset($item['admin']) && !empty($item['admin'])) {
                     $admin = $this->pool->getInstance($item['admin']);
 
-                    // skip menu item if no `list` url is available or user doesn't have the LIST access rights
+                    // Return null if no `list` url is available or user doesn't have the LIST access rights
                     if (!$admin->hasRoute('list') || !$admin->hasAccess('list')) {
-                        continue;
+                        return null;
```

Comment:
There could not be another item in this group because on_top allows only 1 item and if not it throws RuntimeException with message "You can't use "on_top" option with multiple same name groups."